cmd/zpool/zpool_main.c

```diff
@@ -51,6 +51,7 @@
 #include <sys/fm/util.h>
 #include <sys/fm/protocol.h>
 #include <sys/zfs_ioctl.h>
+#include <math.h>
```

Comment:
Not sure I follow.  The niceprint low-level functions (zfs_nicenum/zfs_nicetime/__zfs_nicenum) are in libzfs_util.c, not zpool_main.c.  Are you suggesting moving them out of libzfs_util.c to a new file?